Core/include/Acts/Seeding/HoughTransformUtils.ipp

```diff
@@ -120,12 +149,15 @@ Acts::HoughTransformUtils::PeakFinders::LayerGuidedCombinatoric<
   std::vector<PeakFinders::LayerGuidedCombinatoric<identifier_t>::Maximum>
       maxima;
   // loop over the non empty bins
-  for (auto [x, y] : plane.getNonEmptyBins()) {
+  for (auto nbin : plane.getNonEmptyBins()) {
     // and look for the ones that represent a maximum
-    if (passThreshold(plane, x, y)) {
+
+    std::array<std::size_t, 2> xy = plane.axisBins(nbin);
```

Comment:
if the plane communicates its preferred bins by global index, should `hitIds` and `passThreshold` also work with global bin indices?

Reply:
Would be something for a future development